src/main/scala/codacy/dockerApi/utils/CommandRunner.scala

```diff
@@ -19,15 +19,18 @@ object CommandRunner {
 
     val pio = new ProcessIO(_.close(), readStream(stdout), readStream(stderr))
 
-    val process = Process(cmd, dir).run(pio)
-    val result = Try(process.exitValue())
+    Try(Process(cmd, dir).run(pio)) match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-engine-scala-seed/file/1267284686/issues/source?bid=2836226&fileBranchId=2941544#l22)